qiskit_experiments/calibration_management/calibrations.py

```diff
@@ -1609,18 +1617,18 @@ def __eq__(self, other: "Calibrations") -> bool:
         - The backends have the same name.
         - The backends have the same version.
         - The calibrations contain the same schedules.
-        - The stored paramters have the same values.
+        - The stored parameters have the same values.
         """
         if self.backend_name != other.backend_name:
             return False
 
         if self._backend_version != other.backend_version:
             return False
 
-        # Compare the contents of schedules, schedules are compared by their string
-        # representation because they contain parameters.
         for key, schedule in self._schedules.items():
-            if repr(schedule) != repr(other._schedules.get(key, None)):
+            other_sched = other._schedules.get(key, None)
```

Comment:
One thing to note here -- this method does not test that the parameters are linked across schedules in the same way in `self` and `other`.